### ISSUE TITLE
[Logstash ext] Wrong data sometimes sent to logstash

### ISSUE THREAD

**Issue by bperian** [ntop/ntopng#1160]
Reported by @jevonearth

If you search the block of text for SERVER_NW_LATEN I think you will narrow down the source. It looks like one block of text is getting overwritten with another.

Possibly issue with how data block is sent .

```
[2017-04-13T19:37:26,665][WARN ][logstash.codecs.jsonlines] JSON parse error, original data now in message field {:error=>#<LogStash::Json::ParserError: Unexpected character ('n' (code 110)): was expecting a colon to separate field name and value
 at [Source: { "ntop_timestamp": "2017-04-13T18:20:03.0Z", "IN_SRC_MAC": "00:00:00:00:00:00", "OUT_DST_MAC": "00:00:00:00:00:00", "IPV4_SRC_ADDR": "151.101.0.133", "IPV4_DST_ADDR": "184.70.24.214", "L4_SRC_PORT": 443, "L4_DST_PORT": 35364, "PROTOCOL": 6, "L7_PROTO": 91, "L7_PROTO_NAME": "SSL", "TCP_FLAGS": 18,"IN_PKTS": 22, "IN_BYTES": 9449, "OUT_PKTS": 0, "OUT_BYTES": 0, "FIRST_SWITCHED": 1492107328, "LAST_SWITCHED": 1492107603, "json": { "15": "10.60.58.44", "10": "2", "14": "12", "5": "0", "16": "54113", "17": "6327" }, "CLIENT_NW_LATENCY_MS": 0.000000, **"SERVER_NW_LATEN{ "ntop_timestamp**": "2017-04-13T18:20:06.0Z", "IN_SRC_MAC": "00:00:00:00:00:00", "OUT_DST_MAC": "00:00:00:00:00:00", "IPV4_SRC_ADDR": "10.60.58.5", "IPV4_DST_ADDR": "104.244.46.167", "L4_SRC_PORT": 59408, "L4_DST_PORT": 443, "PROTOCOL": 6, "L7_PROTO": 120, "L7_PROTO_NAME": "SSL.Twitter", "TCP_FLAGS": 18, "IN_PKTS": 40, "IN_BYTES": 3173, "OUT_PKTS": 0, "OUT_BYTES": 0, "FIRST_SWITCHED": 1492107328, "LAST_SWITCHED": 1492107606, "json": { "15": "184.70.24.213", "10": "12", "14": "2", "5": "0", "16": "0", "17":"13414" }, "CLIENT_NW_LATENCY_MS": 0.000000, "SERVER_NW_LATENCY_MS": 0.000000, "DST_IP_COUNTRY": "US", "DST_IP_LOCATION": [ -122.412804, 37.775799 ], "NTOPNG_INSTANCE_NAME": "ntop-redder-dev", "INTERFACE": "tcp:\/\/127.0.0.1:5559c [probeIP: 10.60.58.39]" }; line: 1, column: 572]>, :data=>"{ \"ntop_timestamp\":\"2017-04-13T18:20:03.0Z\", \"IN_SRC_MAC\": \"00:00:00:00:00:00\", \"OUT_DST_MAC\": \"00:00:00:00:00:00\", \"IPV4_SRC_ADDR\": \"151.101.0.133\", \"IPV4_DST_ADDR\": \"184.70.24.214\", \"L4_SRC_PORT\": 443, \"L4_DST_PORT\": 35364, \"PROTOCOL\": 6, \"L7_PROTO\": 91, \"L7_PROTO_NAME\": \"SSL\", \"TCP_FLAGS\": 18, \"IN_PKTS\": 22, \"IN_BYTES\": 9449, \"OUT_PKTS\": 0, \"OUT_BYTES\": 0, \"FIRST_SWITCHED\": 1492107328, \"LAST_SWITCHED\": 1492107603, \"json\": { \"15\": \"10.60.58.44\", \"10\": \"2\", \"14\": \"12\", \"5\": \"0\", \"16\": \"54113\", \"17\": \"6327\" }, \"CLIENT_NW_LATENCY_MS\": 0.000000, \"SERVER_NW_LATEN{ \"ntop_timestamp\": \"2017-04-13T18:20:06.0Z\", \"IN_SRC_MAC\": \"00:00:00:00:00:00\", \"OUT_DST_MAC\": \"00:00:00:00:00:00\", \"IPV4_SRC_ADDR\": \"10.60.58.5\", \"IPV4_DST_ADDR\": \"104.244.46.167\", \"L4_SRC_PORT\": 59408, \"L4_DST_PORT\": 443, \"PROTOCOL\": 6, \"L7_PROTO\": 120, \"L7_PROTO_NAME\": \"SSL.Twitter\", \"TCP_FLAGS\": 18, \"IN_PKTS\": 40, \"IN_BYTES\": 3173, \"OUT_PKTS\": 0, \"OUT_BYTES\": 0, \"FIRST_SWITCHED\": 1492107328, \"LAST_SWITCHED\": 1492107606, \"json\": { \"15\": \"184.70.24.213\", \"10\": \"12\", \"14\": \"2\", \"5\": \"0\", \"16\": \"0\", \"17\": \"13414\" }, \"CLIENT_NW_LATENCY_MS\": 0.000000, \"SERVER_NW_LATENCY_MS\": 0.000000, \"DST_IP_COUNTRY\": \"US\", \"DST_IP_LOCATION\": [ -122.412804, 37.775799 ], \"NTOPNG_INSTANCE_NAME\": \"ntop-redder-dev\", \"INTERFACE\": \"tcp:\\/\\/127.0.0.1:5559c [probeIP: 10.60.58.39]\" }"}



```

**Comment by bperian** [comment]
@jevonearth Issue should be fixed in https://github.com/ntop/ntopng/pull/1161

**Comment by simonemainardi** [comment]
issue is fixed as of #1161
